### DISCUSSION TITLE
Feature request: Disable instruction text after .enable_point_picking

### DISCUSSION THREAD

**Discussion by chessdenman** [pyvista/pyvista#6055] [Q&A] [answered]
### Describe the feature you would like to be added.

Apologies if this is already possible but I can't find out how.
currently enabling point picking in a plotter results in text ("Right click or press P to pick under mouse") at the top of the plot. I would like the option to disable this text.

### Pseudocode or Screenshots

```python
def plotPot(self, points):
    self.polyData = pv.PolyData(points)
    self.polyData.lines = np.hstack([[points.shape[0]], np.arange(points.shape[0])])
    actor = self.plotter.add_mesh(self.polyData, color="grey", point_size=5, render_points_as_spheres=True)
    self.actorsDictionary["potPoints"] = [actor]
    self.plotter.enable_point_picking(callback=self.on_point_picked)
```
<img width="614" alt="Screenshot 2024-05-08 at 20 39 58" src="https://github.com/pyvista/pyvista/assets/21134453/7d964b40-5db7-4bee-b2fa-352eefb45e95">

Or drag your screenshot here!

**Comment by user27182** [comment, marked as answer]
According to the docs for [enable_point_picking](https://docs.pyvista.org/version/stable/api/plotting/_autosummary/pyvista.Plotter.enable_point_picking.html#pyvista-plotter-enable-point-picking), you can use `show_message=False` to disable the message.
